front/src/components/AdminDataGrid.tsx

```diff
@@ -75,95 +75,36 @@ import Link from 'next/link';
     },
   ];
 
-const rows = [
-  {
-    id: 1,
-    level: "easy",
-    firstName: "Jon",
-    age: new Date("2000-02-05"),
-    important: "false",
-    status: "open",
-    message: "frontend",
-  },
-  {
-    id: 2,
-    level: "easy",
-    firstName: "Cersei",
-    age: new Date("2000-02-05"),
-    important: "ture",
-    status: "open",
-    message: "hello",
-  },
-  {
-    id: 3,
-    level: "normal",
-    firstName: "Jaime",
-    age: new Date("2000-02-05"),
-    important: "false",
-    status: "close",
-    message: "frontend",
-  },
-  {
-    id: 4,
-    level: "hard",
-    firstName: "Arya",
-    age: new Date("2000-02-05"),
-    important: "false",
-    status: "open",
-    message: "backend",
-  },
-  {
-    id: 5,
-    level: "hard",
-    firstName: "Daenerys",
-    age: new Date("2000-02-05"),
-    important: "true",
-    status: "close",
-    message: "backend",
-  },
-  {
-    id: 6,
-    level: "easy",
-    firstName: "Aill",
-    age: new Date("2000-02-05"),
-    important: "true",
-    status: "open",
-    message: "hello",
-  },
-  {
-    id: 7,
-    level: "hard",
-    firstName: "Ferrara",
-    age: new Date("2000-02-05"),
-    important: "ture",
-    status: "open",
-    message: "frontend",
-  },
-  {
-    id: 8,
-    level: "normal",
-    firstName: "Rossini",
-    age: new Date("2000-02-05"),
-    important: "true",
-    status: "open",
-    message: "frontend",
-  },
-  {
-    id: 9,
-    level: "hard",
-    firstName: "Harvey",
-    age: new Date("2000-02-05"),
-    important: "false",
-    status: "open",
-    message: "frontend",
-  },
-];
+export default function DataGridDemo() {
+  const [rows , setRows] = React.useState([]);
+  const [snackbar , setSnackbar] = React.useState({open: false, message:'' , severity:'success'});
+  const fetchData = async () =>{
+    try{
+    const response = await axios.get('/api/get/all');
+    setRows(response.data);
```

Comment:
Before setting the data make sure data.age is not string and converted into date using this method : 
```ts
response.data.forEach((row: any) => {
        row.age = new Date(row.age);
      });
```